include/fmt/format-inl.h

```diff
@@ -371,7 +371,7 @@ class fp {
   explicit fp(Double d) {
     // Assume double is in the format [sign][exponent][significand].
     typedef std::numeric_limits<Double> limits;
-    const int double_size = sizeof(Double) * char_size;
+    const size_t double_size = sizeof(Double) * char_size;
```

Comment:
This might cause narrowing warnings below. I suggest static casting to int instead.